docs/docs/develop/advanced-concepts/05-encoding.md

```diff
@@ -152,13 +152,13 @@ message Profile {
 }
 ```
 
-In this `Profile` example, we hardcoded `account` as a `BaseAccount`. However, there are several other types of [user accounts related to vesting](../modules/auth/1-vesting.md), such as `BaseVestingAccount` or `ContinuousVestingAccount`. All of these accounts are different, but they all implement the `AccountI` interface. How would you create a `Profile` that allows all these types of accounts with an `account` field that accepts an `AccountI` interface?
```

Comment:
1-vesting.md doesnt exist as a file? it doesnt seem to exist in the current structure either, maybe we delete it?

Reply:
It is built from the pre.sh on build so wont have its own file